app/controllers/items_controller.rb

```diff
@@ -0,0 +1,79 @@
+class ItemsController < ApplicationController
+  before_action :set_item, only: [:show, :edit, :update]
+
+  def index
+    @item_types = ItemType.all
+    @q = Item.all.where(deleted_at: nil).search(params[:q])
```

Comment:
The `where` will likely have to become a [default scope](http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Default/ClassMethods.html#method-i-default_scope)